packages/x-components/src/main.ts

```diff
@@ -1,18 +1,25 @@
 import 'reflect-metadata';
 import Vue from 'vue';
 import App from './App.vue';
+import { setupDevtools } from './plugins/devtools/devtools.plugin';
 import router from './router';
+import { baseInstallXOptions, baseSnippetConfig } from './views/base-config';
+import { XInstaller } from './x-installer/x-installer/x-installer';
 
 Vue.config.productionTip = false;
-
-const app = new Vue({
-  router,
-  render: h => h(App)
-}).$mount('#app');
-
-// Injecting XComponents into the global window object when Cypress is injected for testing purposes
```

Comment:
We don't need anymore this global object for Cypress?

Reply:
nope, and I think we never did 🤣